docusaurus.config.js

```diff
@@ -26,7 +26,7 @@ const config = {
   // to replace "en" with "zh-Hans".
   i18n: {
     defaultLocale: 'en',
-    locales: ['en', 'de', 'es'],
+    locales: ['en', 'de', 'es', 'zh-Hans'],
     localeConfigs: {
```

Comment:
we actually don't need `localeConfigs` at all, these are the defaults you can just remove this section